melodies_monet/plots/surfplots.py

```diff
@@ -39,8 +40,15 @@ def make_24hr_regulatory(df, col=None):
         dataframe with applied calculation
         
     """
-    return calc_24hr_ave(df, col)
+    #return calc_24hr_ave(df, col)
+    return calc_24hr_ave_v1(df, col)
 
+def calc_24hr_ave_v1(df, col=None):
+    df.index = df.time_local
+    # select sites with nobs >=12
+    df_24hr_ave = (df.groupby("siteid")[col].resample("D").sum(min_count=12)/df.groupby("siteid")[col].resample("D").count()).reset_index().dropna()
```

Comment:
I only found the requirement of 75% completeness for 8-hr ozone calculations. It could also be applied to 24-hr PM2.5 calculation as well. I will update this for PM2.5.